aptos-move/vm-genesis/src/lib.rs

```diff
@@ -272,7 +285,19 @@ pub fn encode_genesis_change_set(
     initialize_randomness_config(&mut session, randomness_config);
     initialize_randomness_resources(&mut session);
     initialize_on_chain_governance(&mut session, genesis_config);
-    create_and_initialize_validators(&mut session, validators);
+
+    create_accounts(&mut session, accounts);
+
+    if validators.len() > 0 {
+        create_and_initialize_validators(&mut session, validators);
```

Comment:
@supra-yoga didn't understand this part. I think we are always connecting every validator with a pool. So why is this `if` statement required? We anyway need to provide `ValidatorConfig` as a parameter for pool creation.

Reply:
This is done to preserve the backwards compatibility in `smr-moonshot` test frameworks. 

By supplying `Validators` here, we specify that we want to start a network with plain stakepool-backed Validators. This is because most of the tests in smr-moonshot only require a executor from a valid network to test functionalities like transaction batch processing, block processing, epoch transition checks etc. These remain agnostic to the type of pools that back the Validators. And also, adding PBO data to the test framework in smr-moonshot induces a large Changeset turns out as a blocker for Round 5 release. 

Though I do agree that we should eventually phase out plain stake-pool backed Validators from genesis transaction encoders and update the test framework in smr-moonshot accordingly.

This debt is now tracked in #46 and is assigned to myself.